ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -269,4 +274,30 @@ static boolean bug1642386Affected(Map<String, String> attributes) {
         && attributes.get(Attribute.APP_VERSION) != null
         && attributes.get(Attribute.APP_VERSION).matches("^([0-9]|[0-2][0-7])\\..*"); // <= 27
   }
+
+  // See bug 1673433 for discussion of affected URIs
+  private static boolean bug1673433Affected(String uri, Map<String, String> attributes) {
+    // See: https://github.com/mozilla-services/cloudops-infra/blob/master/projects/data-ingestion/
+    // k8s/charts/data-ingestion/templates/filter-configmap.yaml#L14-L41
+    if (uri == null) {
+      return false;
+    }
+
+    final ImmutableSet<String> allowedDocTypes = ImmutableSet.of("/main/", "/core/", "/sync/",
+        "/sslreports", "/FirefoxOS/");
+
+    boolean dropPing = true;
+    for (String docType : allowedDocTypes) {
+      if (uri.contains(docType)) {
+        dropPing = false;
+        break;
+      }
+    }
+
+    if (attributes.get(Attribute.ARGS).contains("v=4")) {
+      dropPing = false;
+    }
+
+    return dropPing;
+  }
```

Comment:
This tries to re-implement the logic from https://github.com/mozilla-services/cloudops-infra/blob/master/projects/data-ingestion/k8s/charts/data-ingestion/templates/filter-configmap.yaml#L14-L41

Just to make sure, will the `v=4` URI arg be part of `args` in `attributes`?

Reply:
Yes, I believe this is correct, that we end up with an "args" key in `attributes`.

Reply:
This is indeed documented in https://mozilla.github.io/gcp-ingestion/architecture/edge_service_specification/